base/src/main/java/com/smartdevicelink/proxy/SystemCapabilityManager.java

```diff
@@ -78,14 +80,17 @@
 	private final ISdl callback;
 	private OnRPCListener rpcListener;
 	private boolean shouldConvertDeprecatedDisplayCapabilities;
+	private HMILevel currentHMILevel;
 
 	public SystemCapabilityManager(ISdl callback) {
 		this.callback = callback;
 		this.LISTENER_LOCK = new Object();
 		this.onSystemCapabilityListeners = new HashMap<>();
 		this.cachedSystemCapabilities = new HashMap<>();
 		this.systemCapabilitiesSubscriptionStatus = new HashMap<>();
+		this.systemCapabilitiesSubscriptionStatus.put(SystemCapabilityType.DISPLAYS, true);
 		this.shouldConvertDeprecatedDisplayCapabilities = true;
+		this.currentHMILevel = HMILevel.HMI_NONE;
```

Comment:
Is this what iOS does? IMO our `currentHMILevel` would be none until we are at least registered on the IVI and even then we are inferring that. However, I'm not sure if it makes much of a difference to simply put this as the initial state or not so if it is what iOS does then this comment can be resolved without action.

Reply:
Yes, iOS does a similar thing: https://github.com/smartdevicelink/sdl_ios/blob/4f650efc70d853d31b14ccc3e64210f192f3f9b7/SmartDeviceLink/SDLSystemCapabilityManager.m#L97